### ISSUE TITLE
Nuget install fails on .Net 3.5

### ISSUE THREAD

**Issue by dingjimmy** [ninject/Ninject.Extensions.Wcf#35]
Attempting to install Ninject.Extensions.Wcf 3.2.2 nuget package on a .Net 3.5 WCF service on windows 8.1 It fails with the following error in the package manager console:
> Failed to add reference to 'System.ServiceModel.Activation'. Please make sure that it is in the Global > Assembly Cache.

Tried using version 3.0.0.5 and then come accross this bug: https://github.com/ninject/Ninject.Web.Common/issues/6

The full package manager output is below...
```
Attempting to gather dependencies information for package 'Ninject.Extensions.Wcf.3.2.2' with respect to project 'InstrumentServer\Optimal.SynTQ.InstrumentServer.Kaiser.Services', targeting '.NETFramework,Version=v3.5'
Attempting to resolve dependencies for package 'Ninject.Extensions.Wcf.3.2.2' with DependencyBehavior 'Lowest'
Resolving actions to install package 'Ninject.Extensions.Wcf.3.2.2'
Resolved actions to install package 'Ninject.Extensions.Wcf.3.2.2'
Adding package 'Ninject.Extensions.Wcf.3.2.2' to folder 'C:\Projects\SynTQ\dev\packages'
Added package 'Ninject.Extensions.Wcf.3.2.2' to folder 'C:\Projects\SynTQ\dev\packages'
Install failed. Rolling back...
Package 'Ninject.Extensions.Wcf 3.2.2' does not exist in project 'Optimal.SynTQ.InstrumentServer.Kaiser.Services'
Removing package 'Ninject.Extensions.Wcf 3.2.2' from folder 'C:\Projects\SynTQ\dev\packages'
Removed package 'Ninject.Extensions.Wcf 3.2.2' from folder 'C:\Projects\SynTQ\dev\packages'
Failed to add reference to 'System.ServiceModel.Activation'. Please make sure that it is in the Global Assembly Cache.
========== Finished ==========

```

**Comment by dingjimmy** [comment]
Investigated this a bit. Looks like Microsoft moved the `System.ServiceModel.Activation` namespace to from *System.ServiceModel.dll* in dotnet35 to *System.ServiceModel.Activation.dll* in dotnet40

**Comment by dingjimmy** [comment]
This issue is fixed in pull request #37
